source/js/buyers-guide/analytics-events.js

```diff
@@ -8,52 +8,16 @@ function getQuerySelectorEvents(pageTitle, productName) {
       action: `donate tap`,
       label: `${pageTitle} donate header`
     },
-    "#donate-button-main": {
+    "#donate-header-btn": {
       category: `buyersguide`,
       action: `donate tap`,
       label: `${pageTitle} donate header`
     },
-    "#donate-button-footer": {
+    ".donate-banner a.btn.btn-secondary": {
       category: `buyersguide`,
       action: `donate tap`,
       label: `${pageTitle} donate footer`
     },
-    "#nav-social-button-fb": {
```

Comment:
we removed this functionality a while back and never removed these query selectors